lib/templates/pullRequestOpened.hbs

```diff
@@ -1,36 +1,36 @@
-{{#if claEnabled}}
-{{#if errorCla}}
-@{{ userName }}, thanks for the pull request!
+{{#if askAboutContributors}} 
+Thank you so much for the pull request @{{ userName }}! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!
 
-If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request.
+The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.
 
-Maintainers, there was an error processing the CLA, resolve it to continue CLA checking.
-```
-{{ errorCla }}
-```
-{{else}}
-{{#if askForCla}}
-Welcome to the Cesium community @{{ userName }}!
+Let's see if this is ready for review.
 
-Can you please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) so that we can review and merge this pull request?
+* ❌ Missing `CONTRIBUTORS.md` entry.
+  * Please add yourself to the [contributors]({{ contributorsUrl }}) file! 
 {{else}}
-@{{ userName }}, thanks for the pull request! Maintainers, we have a signed CLA from @{{ userName }}, so you can review this at any time.
-{{/if}}
+Thanks for the pull request @{{ userName }}! Let's see if this is ready for review.
```

Comment:
To minimize this a little further, I think we should remove `Let's see if this is ready for review.` Here and above.